yesod-test/test/main.hs

```diff
@@ -140,6 +140,18 @@ main = hspec $ do
                 htmlAnyContain "p" "World"
                 htmlAnyContain "p" "Moon"
                 htmlNoneContain "p" "Sun"
+            yit "CSRF token" $ do
```

Comment:
All of these test descriptions are terse and yours follows that pattern, but I think it would be much clearer to mention that the test is looking up a CSRF token with a CSS query. As it is, it's hard to tell at a glance how this test is different from `yit "labels"` above